frdrpc/rpcserver.go

```diff
@@ -45,6 +45,15 @@ var (
 	// set this to 200MiB atm.
 	maxMsgRecvSize = grpc.MaxCallRecvMsgSize(1 * 1024 * 1024 * 200)
```

Comment:
Just catched my eye: I guess with pagination we probably don't need this large msg size anymore.